rules/default.js

```diff
@@ -3,10 +3,23 @@ define([
 ], function($) {
 
   return {
-    'required': function(value) {
+    required: function (value, input, ruleArgs) {
+      if ($(input).attr("type") === "radio") {
+        return Boolean(this._required_radio(input));
+      }
       return Boolean(value);
     },
-    'email': function(value) {
+    _required_radio: function (radio) {
```

Comment:
if this is intended to be a private function you should define it outside the response object